job_templates/ci_job.xml.em

```diff
@@ -183,7 +183,11 @@ docker build --build-arg PLATFORM=arm -t ros2_batch_ci_armhf linux_docker_resour
 sed -i 's+^FROM.*$+FROM osrf/ubuntu_arm64:xenial+' linux_docker_resources/Dockerfile
 docker build --build-arg PLATFORM=arm -t ros2_batch_ci_aarch64 linux_docker_resources
 @[elif os_name == 'linux']@
+@[if turtlebot_demo]@
+docker build --build-arg INSTALL_TURTLEBOT2_DEMO_DEPS=true -t ros2_batch_ci_turtlebot_demo linux_docker_resources
+@[else]@
 docker build -t ros2_batch_ci linux_docker_resources
+@[end if]@
```

Comment:
I definitely can. Is that kind of indentation a common pattern in empy templates/